### ISSUE TITLE
Compilation failed with poppler-0.62

### ISSUE THREAD

**Issue by LazyKent** [jwilk/pdf2djvu#126]
Compilation failed with poppler-0.62.

```
g++ -fopenmp -fomit-frame-pointer -fmessage-length=0 -grecord-gcc-switches -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -Wall -Wempty-body -Werror=overloaded-virtual -Werror=narrowing -Werror=misleading-indentation -pthread -I/usr/include/poppler -I/usr/include/GraphicsMagick  -I/usr/include/uuid   -c -o pdf-unicode.o pdf-unicode.cc
pdf-unicode.cc:29:10: fatal error: UTF8.h: No such file or directory
```

**Comment by jwilk** [comment]
Should be fixed in e263c43bbd552a771f74f0f93ce9cca8b1aa8702.

I'll make a new release soon.

**Comment by jwilk** [comment]
> I'll make a new release soon.

[Done](https://groups.io/g/pdf2djvu/message/85)!
